src/components/Layout/header/header.js

```diff
@@ -51,10 +52,17 @@ export function Header() {
           ))}
         </div>
         <div className="hidden lg:flex lg:flex-1 lg:justify-end">
-          <Link href="#" className="text-sm font-semibold leading-6 text-white">
-            Log in
-            {' '}
-            <span aria-hidden="true">&rarr;</span>
+          <Link
+            href="/login"
+            className="text-sm font-semibold leading-6 mx-3 text-white"
+          >
+            Log in <span aria-hidden="true">&rarr;</span>
```

Comment:
whats that can we just delete it? same for below

Reply:
It deleted.